src/MongoDB.EntityFrameworkCore/Serializers/SerializationHelper.cs

```diff
@@ -103,19 +113,26 @@ private static IBsonSerializer CreateTypeSerializer(Type type, IReadOnlyProperty
             _ when type == typeof(short) => new Int16Serializer(),
             _ when type == typeof(int) => Int32Serializer.Instance,
             _ when type == typeof(long) => Int64Serializer.Instance,
-            _ when type == typeof(ObjectId) => ObjectIdSerializer.Instance,
             _ when type == typeof(TimeSpan) => new TimeSpanSerializer(),
             _ when type == typeof(sbyte) => new SByteSerializer(),
             _ when type == typeof(float) => new SingleSerializer(),
             _ when type == typeof(string) => new StringSerializer(),
             _ when type == typeof(ushort) => new UInt16Serializer(),
             _ when type == typeof(uint) => new UInt32Serializer(),
             _ when type == typeof(ulong) => new UInt64Serializer(),
-            _ when type == typeof(Decimal128) => new Decimal128Serializer(),
             _ when type.IsEnum => EnumSerializer.Create(type),
+
+            // Nullable
             {IsGenericType: true} when type.GetGenericTypeDefinition() == typeof(Nullable<>)
                 => CreateNullableSerializer(type.GetGenericArguments()[0]),
+
+            // Mongo structs
+            _ when type == typeof(ObjectId) => ObjectIdSerializer.Instance,
+            _ when type == typeof(Decimal128) => new Decimal128Serializer(),
```

Comment:
We should configure serializers for ObjectId and Decimal128, but this seems unrelated to the PR at hand. Move to a separate PR?

Reply:
This doesn't add/remove any support, it just groups them together with comments as the list was getting long. I can revert.

Reply:
Removed this clean-up from the PR.